### ISSUE TITLE
Get Audio and Video Toggle Controls Back

### ISSUE THREAD

**Issue by Static2600** [OpenShot/openshot-qt#1202]
Core i7 7th Gen, 16GB RAM, SSD, Intel Graphics
Ubuntu 17.10, using xorg
OpenShot Version: 2.4.1 (appimage version)

I'm fairly new to OpenShot and was using the version in the Ubuntu repo until this morning. It was the 1.4.3 version if I recall correctly. Under that version there were handy toggles on each track and each clip to enable or disable audio and video. Is there a way to get these back under 2.4.1?  If not is there another easy way to enable / disable video? Right clicking and using the properties for each clip is very tedious.

Thanks!

PS - This is what I'm referring to ... http://www.openshotusers.com/help/1.3/en/ar01s11.html

**Comment by peanutbutterandcrackers** [comment]
@DylanC - Give this man an 'enhancement' label cookie, please? 😄

**Comment by peanutbutterandcrackers** [comment]
@Static2600 - Yes, it'd be really handy to have those controls back, though. :)

**Comment by mjtorn** [comment]
Yeah, this is (or was) a most useful feature, still missing in 2.4.3. As this issue has been open for nearly a year, any news on getting it back?

**Comment by mikelacsa** [comment]
The same enhancement was requested in [1260](https://github.com/OpenShot/openshot-qt/issues/1260)
